userspace/libscap/engine/gvisor/scap_gvisor.cpp

```diff
@@ -613,4 +639,19 @@ engine::runsc_result engine::runsc_trace_delete(const std::string &session_name,
 	return runsc((char **)argv);
 }
 
+engine::runsc_result engine::runsc_trace_procfs(const std::string &sandbox_id)
```

Comment:
I am starting to thinking: what if we move all the `runsc_...` operations under the `runsc` namespace?
I think it would feel better!
Super nit-picking here though, feel free to skip this!

Reply:
I was thinking about that too from the very beginning :) Let me give it a try and see what it will look like!